functions/expression/1.0/index.js

```diff
@@ -1,4 +1,4 @@
-import templayed from 'templayed';
```

Comment:
Just curious, do we want to limit expressions to a single function block? Or have it in a shared directory?

Reply:
If we gonna implement other blocks with this templated library then we should move it imo

Reply:
What @emiel-betty says indeed